metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -456,11 +462,27 @@ def get_custom_properties(node: DBTNode) -> Dict[str, str]:
     return custom_properties
 
 
+def match_target_platform_instance(
```

Comment:
lets call this `get_entity_platform_instance`, since it may be getting a platform instance for a dbt node and not a target node

Reply:
done